packages/text-editor/src/components/TextEditor.svelte

```diff
@@ -50,7 +51,7 @@
     if (editor) editor.setEditable(editable)
   }
   export function submit (): void {
-    if (!editor.isEmpty) {
+    if (!editor.isEmpty || haveAttachment) {
       content = editor.getHTML()
       dispatch('content', content)
```

Comment:
Let's better send submit event with {isEmpty: boolean, content: content} and handle it properly in all use places.

Reply:
We already have isEmpty, so just handle empty submit inplace.